holoviews/util/__init__.py

```diff
@@ -339,6 +339,78 @@ def __call__(self, *args, **params):
         Store.current_backend = selected_backend
 
 
+def save(obj, filename, fmt='auto', backend=None, **kwargs):
```

Comment:
To me the concept of "saving" is something that could later be loaded in to restore the state, so I personally would call this function `export`, which to me conveys saving it in a separate, non-native format from which the state can't normally be recovered.  I know we already have exporting in HoloViews, but from what I can see that's just about exporting archives, which I don't think conflicts with this function's meaning in any way -- both of them do involve exporting to non-native formats (though archive exporting also includes an "archiving" aspect that is like saving, i.e., allows restoration of the original objects).

Reply:
I would prefer not to change the terminology, it's ``hv.renderer('bokeh').save`` and both matplotlib (``savefig``) and bokeh (``save``) use the same terminology.

Reply:
Sigh.  Ok.

Reply:
Sigh!